jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java

```diff
@@ -76,11 +77,14 @@ private void add(TarArchiveEntry tarArchiveEntry) throws IOException {
       if (namePath.getParent() != namePath.getRoot()) {
         Path tarArchiveParentDir = Verify.verifyNotNull(namePath.getParent());
         TarArchiveEntry dir = new TarArchiveEntry(DIRECTORY_FILE, tarArchiveParentDir.toString());
-        dir.setModTime(FileEntriesLayer.DEFAULT_MODIFICATION_TIME.toEpochMilli());
         dir.setUserId(0);
         dir.setGroupId(0);
         dir.setUserName("");
         dir.setGroupName("");
+        // In this particular place we have to use default modification time (1 second past epoch)
+        // instead the modification time user might've set in JIB settings. That's because we have
+        // no way to find out the modification time user might've set in JIB settings here.
```

Comment:
This is not true. Jib intentionally doesn't allow setting the mod time of directories. Let's just remove the comment.